test/performance/service_creation_test.go

```diff
@@ -0,0 +1,106 @@
+// +build performance
+
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package performance
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/knative/pkg/test/logging"
+	"github.com/knative/serving/pkg/reconciler"
+	"github.com/knative/serving/test"
+	"github.com/knative/test-infra/shared/prometheus"
+	"github.com/knative/test-infra/shared/testgrid"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	tCreateServiceN    = "TestCreateService"
+	tCreate20ServicesN = "TestCreate20Services"
```

Comment:
perhaps, tCreateNServicesN = "TestCreate%dServices" and then below fmt.Sprintf().
Since if tomorrow we decide to change 20 to 25, it'll be more places to change, perhaps missing this one?

Reply:
Done